### ISSUE TITLE
Western blotting - Change the error message regarding 15 samples

### ISSUE THREAD

**Issue by ald32** [starteam/starcellbio_html#310]
After selecting 15 samples on the Western blotting sample prep page (in Exercise 3), I encountered an message saying that I had selected more than 15 samples and that only 15 samples are allowed (and that we recommend that you select 14 so you can add the marker). This message should only appear if more than 15 samples are selected.
In addition, the " mark should be removed at the end of the message.

**Comment by ald32** [comment]
I just added 15 samples to my western blotting exp technique, but then it still let me add the protein marker even though there isn't any available lane for the marker. This then created problems because the colorful marker bands appeared off the gel once I selected 'Run gel & transfer'. 

If 15 samples are selected on the sample prep page, then the Marker button shouldn't appear in the samples window so students can't even add it to their gels. Also, the colorful bands after selecting 'Run & Transfer' wouldn't appear and the horizontal lines on the side of the blot wouldn't appear.
